packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -157,14 +176,15 @@ export async function generateOutputDirectory(
   if (outputBundleOptions.needsServerGenerated) {
     await generateServer(outputBundleOptions);
   }
+  if (!angularVersion) throw new Error("Could not find the angular version of the application");
```

Comment:
Actually, why does `angularVersion` need to be nullable here at all?

Reply:
I think process.env.FRAMEWORK_VERSION is nullable by default, and I passed that in build.ts generateOutputDirectory(root, outputBundleOptions, process.env.FRAMEWORK_VERSION). So asserting in here, is it better to assert in build.ts?

Reply:
Moved the assert to build.ts, makes more sense to assert `process.env.FRAMEWORK_VERSION` right before using it